### ISSUE TITLE
vagrant setup script produces error in Leap 15.1

### ISSUE THREAD

**Issue by ChaseCrum** [sigsteve/vagrant-caasp#42]
I encountered the error below and was asked to report an issue. I am using Leap 15.1

when executing sudo ./libvirt_setup/openSUSE_vagrant_setup.sh, I get the following error:
chasecrum@linux-yblm:~/code/vagrant-caasp>  sudo ./libvirt_setup/openSUSE_vagrant_setup.sh
+ type rpmdev-vercmp
++ rpm -qa zypper
+ rpmdev-vercmp zypper-1.14.4 zypper-1.14.27-lp151.1.2.x86_64
WARNING: hyphen in release2: 1.14.27-lp151.1.2.x86_64

rpmdev-vercmp <epoch1> <ver1> <release1> <epoch2> <ver2> <release2>
rpmdev-vercmp <EVR1> <EVR2>
rpmdev-vercmp # with no arguments, prompt

Exit status is 0 if the EVR's are equal, 11 if EVR1 is newer, and 12 if EVR2
is newer.  Other exit statuses indicate problems.

As a work around, this part of the script was commented out and the following command worked:

zypper --no-gpg-checks in -y https://releases.hashicorp.com/vagrant/2.2.5/vagrant_2.2.5_x86_64.rpm

After this command successfully completed, the vagrant install script (with commented out section) ran successfully to completion.

**Comment by SweBarre** [comment]
strange..
"it works on my computer"™ :)

```
$ cat /etc/os-release 
NAME="openSUSE Leap"
VERSION="15.1 "
ID="opensuse-leap"
ID_LIKE="suse opensuse"
VERSION_ID="15.1"
PRETTY_NAME="openSUSE Leap 15.1"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:leap:15.1"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"

$ uname -a
Linux exsilio 4.12.14-lp151.28.13-default #1 SMP Wed Aug 7 07:20:16 UTC 2019 (0c09ad2) x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by ne0777** [comment]
@sigsteve - I believe this can be closed due to the #53 PR.

**Comment by ne0777** [comment]
Closing with #53
